### PR TITLE
velero migration plugin implementation

### PR DESCRIPTION
**Pull request by sseago**
At this point, everything builds without error but has not yet been tested.

### CODE REVIEW

**Comment by sseago** [comment]
I've tested this with the controller and a migration that tests every plugin except for the pv/pvc-related and quiesce-related ones it works fine. The PR will need an update after the general plugin is pushed to update the Gopkg.toml reference to point to the fusor repo rather than the PR branch on my fork.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM alpine:3.6
+RUN mkdir /plugins
+ADD velero-* /plugins/
+USER nobody:nobody
+ENTRYPOINT ["/bin/ash", "-c", "cp /plugins/* /target/."]
```
Comment (pranavgaikwad) [inline Dockerfile:19]: Looks like a typo `/bin/bash`

Comment (pranavgaikwad) [reply Dockerfile:19]: My bad. Didn't notice alpine image.

**Review by djwhatle — APPROVED** [review]
ACK

**Pull request merged**